tests/kafkatest/services/mirror_maker.py

```diff
@@ -98,6 +99,7 @@ def __init__(self, context, num_nodes, source, target, whitelist=None, blacklist
         self.blacklist = blacklist
         self.source = source
         self.target = target
+        self.offsets_storage = offsets_storage
```

Comment:
And more generally -- I'd think of these service classes as similar to public APIs. Don't trust the inputs you're being sent. Expect people to use them in weird ways. Once you provide support for something, there are probably other tests you're not thinking of that are dependent on them, so be wary of incompatible changes. Etc.